### PR TITLE
Upgrade versions

### LINKED ISSUE #10: Update to latest SAFE Stack technologies
So .NET 6, Fable 3.7 etc.

### PR DESCRIPTION
**Pull request by isaacabraham**
* Upgrade to .NET 6, Giraffe 6 etc. and Fable 3.7
* Fixes some logging
* Makes transaction ingestion work in batches to avoid completely swamping the CPU / network.

Closes #10

### CODE REVIEW

**Comment by martinbryant** [comment]
@isaacabraham I have updated to a point where `dotnet run Azure` correctly deploys the resources but azcopy(v10) runs and says it was successful in transferring the json data from the blob storage to the newly created table storage. However I cannot see any of the table data. Could use an assist on this.

Screenshot 2023-12-21 at 16 42 49

**Comment by martinbryant** [comment]
Issues left to sort

- [ ] We can no longer use azcopy to seed data into the Table Storage inside `dotnet run Azure`. We need to find an alternative way to do this. azcopy referenced in the current README is Windows only and marked as deprecated.
- [ ] Feliz AgGrid is not using the current ag-grd-react and ag-grid-community packages and there are issues which I have referenced in https://github.com/CompositionalIT/feliz-ag-grid/issues/16

**Comment by martinbryant** [comment]
The project compiles ok but Rider/VsCode is complaining it cannot find the React.Element type so intellisense is not working correctly in the Client - The reason is a clash between ReactElement types in packages using different versions of Feliz, v1 vs v2.

We need to import new versions of

- [ ] Feliz.Tippy
- [ ] Feliz.ReactSkeleton

when they have been updated to use Feliz 2

File: Build.fs [hunk 1]
```diff
@@ -49,7 +49,7 @@ Target.create "Azure" (fun _ ->
         always_on
         sku WebApp.Sku.B1
         operating_system Linux
-        runtime_stack Runtime.DotNet50
+        runtime_stack (Runtime.DotNet "8.0")
```
Comment (isaacabraham) [inline Build.fs:52]: Change this to use strong typing

**Pull request merged**